examples/tensorflow/local/horovod_keras_mnist.py

```diff
@@ -0,0 +1,151 @@
+"""
+This script is a simple MNIST training script which uses Horovod and Tensorflow's Keras interface.
+It has been orchestrated with SageMaker Debugger hook to allow saving tensors during training.
+Here, the hook has been created using its constructor to allow running this locally for your experimentation.
+When you want to run this script in SageMaker, it is recommended to create the hook from json file.
+Please see scripts in either 'sagemaker_byoc' or 'sagemaker_official_container' folder based on your use case.
+"""
+
+# Standard Library
+import argparse
+import math
+import os
+
+# Third Party
+import horovod.tensorflow.keras as hvd
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import backend as K
+from tensorflow.keras.datasets import mnist
+from tensorflow.keras.layers import Conv2D, Dense, Dropout, Flatten, MaxPooling2D
+from tensorflow.keras.models import Sequential
+
+# First Party
+import smdebug.tensorflow as smd
+
+
+def str2bool(v):
+    if isinstance(v, bool):
+        return v
+    if v.lower() in ("yes", "true", "t", "y", "1"):
+        return True
+    elif v.lower() in ("no", "false", "f", "n", "0"):
+        return False
+    else:
+        raise argparse.ArgumentTypeError("Boolean value expected.")
```

Comment:
I've seen this method separately in a lot of scripts. What do you think the best way to refactor it out is?

Reply:
Unfortunately argparse doesn't have good support for bools. I use this method from here: https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse

Yeah the duplication came to my head as I was adding this too. We could add it to a smdebug utils function